bgpd/bgp_vty.c

```diff
@@ -6588,6 +6588,62 @@ DEFPY (bgp_imexport_vpn,
 	return CMD_SUCCESS;
 }
 
+DEFPY (af_routetarget_import,
+       af_routetarget_import_cmd,
+       "[no] <rt|route-target> redirect import RTLIST...",
+       NO_STR
+       "Specify route target list\n"
+       "Specify route target list\n"
+       "For redirected traffic matching that import rt\n"
```

Comment:
How about adding something that points to flow-spec?, e.g., "Flow-spec redirect type route target"

Reply:
ok